src-vue/src/views/RepairView.vue

```diff
@@ -78,33 +68,22 @@ export default defineComponent({
             } as GameInstall;
 
             // Send notification telling the user to wait for the process to finish
-            const notification = ElNotification({
-                title: 'Force reinstalling Northstar',
-                message: 'Please wait',
-                duration: 0,
-                type: 'info',
-                position: 'bottom-right'
-            });
+            const notification = showNotification(
+                'Force reinstalling Northstar',
+                'Please wait',
```

Comment:
Looks like we forgot to localise that. I'll make an issue to address it in a separate PR ^^